rts/motoko-rts/src/gc/mark_compact/mark_stack.rs

```diff
@@ -13,13 +13,13 @@ pub const INIT_STACK_SIZE: Words<u32> = Words(64);
 static mut STACK_BLOB_PTR: *mut Blob = null_mut();
```

Comment:
Keeping that one private deliberately?

Reply:
It's not used in tests so no need to make it `pub`.  Should I make it `pub` for consistency?